operator_ui/package.json

```diff
@@ -48,6 +48,7 @@
     "json-pretty-html": "^1.1.2",
     "local-storage-fallback": "^4.1.1",
     "mock-local-storage": "^1.0.5",
+    "moment": "^2.24.0",
```

Comment:
@NavyAdmiral did you consider [`toLocaleDateString`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString)? We should be able to achieve the same thing without this dependency.

The timezone name may be a little different as it includes the number. But we could probably just add that to the end of the strong and not include it as a display option in the call to `toLocaleDateString`

Reply:
@rupurt Initially when I took a look at the story that's what I had in mind. However, since the explorer story looked so much alike (if not identical) to this story, I just replicated the work that's been done there to here. Could you explain how is this operator ui story different from the explorer story?

Reply:
Fair call on keeping it consistent. Lets disable the tooltip also

Reply:
Removed the tooltips. Let's make sure with @se3000 we're on right track with these changes

Reply:
Looks good to me. Wondering if it would be worth pulling out a helper, since the logic of `createdAt && moment(createdAt).format()` is duplicated many places.

Reply:
Extracted it to a helper. 
On the fence if `<TimeAgo>{r.createdAt}</TimeAgo> ({localizedTimestamp(r.createdAt)})` should be extracted to a component. Probably not?